### DISCUSSION TITLE
force RTL to be disabled for some elements

### DISCUSSION THREAD

**Discussion by ehsanonline** [mantinedev/mantine#1108] [Feature requests]
Some fields like email or phone number should be always left-to-right, is there a way to make RTL disable for some part of the app?

**Comment by ehsanonline** [comment]
found a way, using `/* @noflip */` in CSS attributes.

**Comment by rtivital** [comment]
You can wrap parts of your application that require different settings with MantineProvider that does not inherit parent settings – https://mantine.dev/theming/mantine-provider/#nested-mantineproviders
